posts/lets_talk_about_9_to_5.md

```diff
@@ -0,0 +1,19 @@
+
+For me it all comes back to the idea that symptoms and illnesses are inextricably linked but while you can detect illnesses via their symptoms, you cure them by addressing the underlying sickness.
+
+In this case work hours are the symptom and the illness is a teams effectiveness or delivery capability. In an environment where there are flexible working hour policies, i.e. choose your own hours, teams frequently working less than 8 hours are likely to be under-challenged. The hours here are the symptom and you're unlikely to solve the problem by enforcing work hours.
+
+Why is a team likely under-challenged if they regularly work less than 8 hours a week? My current answer to this is because we have a social norm of what office hours look like. Regardless of your flexible work hour policy, the rest of your organisation is likely working an 8 hour day which means there will be some level of social pressure around that 8 hour window. If a team is being challenged, they'll usually fill that socially normal window of 8 hours at which point they feel like they've put enough into their day and unless there's an emergency they'll likely feel justified in going home.
+
+A team that is under-challenged won't need to use the full 8 hours and won't feel pressured to stay for the full 8 hours because their workload does not demand that they fully realise a full working day.
+
+I'd be arguing that a team not filling 6 hours if 6 hours was a more socially normal office day is a smell too, by the way. It's about the combination of pushing yourself to your limit in order to meet the challenge of your workload. Social norms simply make the acceptable time to call it a day at 8 hours which is why I argue that it's a valid symptom or smell to watch out for.
+
+What does healthy look like? I think a team that is challenged will certainly be working a full 8 hours at least once every 2 weeks if they follow a 2 week iteration cycle. If they've challenged themselves during that iteration they'll naturally need to push hard towards the end to ensure that their showcase goes well. There's likely an issue or two if this rarely happens.
+
+What causes under-challenged teams? Lack of pressure from customers for the team to deliver is a big cause of this. A startup spending their investment money on a software engineering team to help them get to market will naturally be very invested in the team challenging themselves to deliver the most value possible. In larger organisations where things generally take longer there can often be a lack of a Product Owner really pushing a team to deliver. Addressing this issue often requires the team to create pressure for themselves, especially when working with an outside organisation.
```

Comment:
causes are 👍 Should we also explore more solutions? and have a paragraph on this topic?